tasks/test.py

```diff
@@ -548,6 +549,16 @@ def lint_python(ctx):
     ctx.run("vulture --ignore-decorators @task --ignore-names 'test_*,Test*' tasks")
 
 
+@task
+def lint_copyrights(_):
+    """
+    Checks that all files matching the search pattern (default: '**/*.go')
+    contain the appropriate copyright header.
```

Comment:
Guessing this used to have a 'search pattern' parameter and it doesn't anymore?
```suggestion
    Checks that all Go files contain the appropriate copyright header.
```

Reply:
It's the other way around. For now this only checks Golang files but we may expand it to do the others types like Python and C. I'll apply your suggestion for now though.

Reply:
ah, right, thanks for explaining!